readme.md

```diff
@@ -26,7 +37,8 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+  When a request comes in to the page ajax sends the information in the form of json
+  which then gets translated back to html/css or whatever it needs to be
```

Comment:
This is close, but somewhat misleading.  Ajax does not respond to a request that comes in to the page.  It does send information in ht form of son, but it is the son response, from the server, that is usually discussed when describing ajax.  It is this response from the (remote) server that we translate into html/css or whatever, not the information that ajax is sending. 
Please review the answer key in the solution branch.